activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -71,7 +71,7 @@ def camelize(term, uppercase_first_letter = true)
       if !uppercase_first_letter || uppercase_first_letter == :lower
         string = string.sub(inflections.acronyms_camelize_regex) { |match| match.downcase! || match }
       elsif string.match?(/\A[a-z\d]*\z/)
-        return inflections.acronyms[string] || string.capitalize
+        return inflections.acronyms[string].try(&:dup) || string.capitalize
```

Comment:
```suggestion
        return inflections.acronyms[string]&.dup || string.capitalize
```

`try` rescue `NoMethodError`, that's not what we want here, especially since `nil.dup == nil`

Reply:
Ahh yep. Thanks @byroot I fixed that.